fixtures/mega-led-lighting/fog-machine.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Fog Machine",
+  "categories": ["Smoke", "Hazer"],
+  "meta": {
+    "authors": ["Ale Allstyle"],
+    "createDate": "2023-05-02",
+    "lastModifyDate": "2023-05-02"
+  },
+  "links": {
+    "productPage": [
+      "http://www.mjledlighting.com"
+    ]
+  },
```

Comment:
Missing manual  and video. Product page goes to main site and I cannot find any fog machines on their site